.github/workflows/electron-linux.yml

```diff
@@ -0,0 +1,53 @@
+name: Electron Rebuild CI - Linux (x32 & x64)
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        electron:
+          - 5
+          - 6
+          - 7
+          - 8
+        include:
+          # includes a new variable for each electron (essentially a switch statement) 
+          - electron: 5
+            nodeVersion: 70
+          - electron: 6
+            nodeVersion: 73
+          - electron: 7
+            nodeVersion: 75
+          - electron: 8
+            nodeVersion: 76
+        arch:
+          - ia32
```

Comment:
I'd probably drop this, since official x86 Node support dropped a couple of versions back

Reply:
Just THANK YOU!!! Having real problems with linux